### DISCUSSION TITLE
Locast.org Quit

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#242] [Show and tell]
Email from Locast.

"We are suspending operations, effective immediately

As a non-profit, Locast was designed from the very beginning to operate in accordance with the strict letter of the law, but in response to the court’s recent rulings, with which we respectfully disagree, we are hereby suspending operations, effective immediately."

Haven't read the ruling. Things are changing in the U.S. Not sure how this will affect the IPTV list projects.

**Comment by Dum4G** [comment]
We're complying to DMCAs. If something persists in the list means nobody cares enough to ban it.
